process/sync/baseSync.go

```diff
@@ -449,10 +449,16 @@ func (boot *baseBootstrap) waitForHeaderHash() error {
 	}
 }
 
-// ShouldSync method returns the synch state of the node. If it returns 'true', this means that the node
+// ShouldSync method returns the sync state of the node. If it returns 'true', this means that the node
 // is not synchronized yet and it has to continue the bootstrapping mechanism, otherwise the node is already
-// synched and it can participate to the consensus, if it is in the jobDone group of this rounder
+// synced and it can participate to the consensus, if it is in the jobDone group of this rounder
+// should note that when the node is not connected to the network, ShouldSync returns true but the SyncBlock
```

Comment:
This is a new idea, so you can capitalize it for readability reasons

Reply:
Done